hloc/colmap_from_opensfm.py

```diff
@@ -0,0 +1,171 @@
+import argparse
+import csv
+import json
+import math
+import typing
+from pathlib import Path
+
+import numpy as np
+from tqdm import tqdm
+
+from hloc import logger
+from hloc.utils.read_write_model import (
+    CAMERA_MODEL_NAMES,
+    Camera,
+    Image,
+    Point3D,
+    write_model,
+)
+
+
+def angle_axis_to_quaternion(angle_axis: np.ndarray) -> typing.List[float]:
```

Comment:
Nit: If we're using python 3.9 or later I think we should use the new style of typing which would be: `list[float]` instead of `typing.List[float]`

Reply:
I didn't know that. Thanks for the info.
Adopted your suggestion. One import less.